### PR TITLE
Update var to const in docs/Web/JavaScript/Reference/Global_Objects/weakest/has

### PR DESCRIPTION
**Pull request by samantharamon**
…eakset/has

#### Summary
Update var to const.

#### Motivation
- Write the Docs 2022 Writing Day project.
- Conform to current specifications.
 
#### Supporting details
None.

#### Related issues
N/A

#### Metadata

This PR…

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [X] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/javascript/reference/global_objects/weakset/has/index.md [hunk 1]
```diff
@@ -38,8 +38,8 @@ has(value)
 ### Using the has method
 
 ```js
-var ws = new WeakSet();
-var obj = {};
+const ws = new WeakSet();
+const obj = {};
 ws.add(window);
 
 mySet.has(window);  // returns true
```
Comment (wbamberg) [inline files/en-us/web/javascript/reference/global_objects/weakset/has/index.md:45]: Although this is not a problem you've introduced, I think it is worth fixing while you are here. We declare the `WeakSet` as `ws` but then try to refer to it as `mySet` which will not work.

I think to align with the other examples we should use `ws` at lines 45 and 46. What do you think?

**Review by wbamberg — COMMENTED** [review]
I had one comment about this example: although it's not a problem your PR introduced I think it is worth fixing.

File: files/en-us/web/javascript/reference/global_objects/weakset/has/index.md (hunk 1 above)
Comment (samantharamon) [reply files/en-us/web/javascript/reference/global_objects/weakset/has/index.md:45]: Totally agree with your proposed change, @wbamberg. Thank you for catching that, as I completely missed it! Making the change now.

Comment (samantharamon) [inline files/en-us/web/javascript/reference/global_objects/weakset/has/index.md:45]:
```suggestion
ws.has(window);  // returns true
```

File: files/en-us/web/javascript/reference/global_objects/weakset/has/index.md [hunk 2]
```diff
@@ -38,8 +38,8 @@ has(value)
 ### Using the has method
```
Comment (OnkarRuikar) [inline files/en-us/web/javascript/reference/global_objects/weakset/has/index.md:38]:
```suggestion
### Using the `has()` method
```
Just a suggestion, highlighting the method name avoids this being flagged as a grammar error.

Comment (samantharamon) [reply files/en-us/web/javascript/reference/global_objects/weakset/has/index.md:38]: Thanks, @OnkarRuikar! That's a great suggestion. Committing it now.

**Review by teoli2003 — APPROVED** [review]
👍

**Pull request merged**